src/update_agent/actor.rs

```diff
@@ -9,13 +9,19 @@ use libsystemd::daemon::*;
 use log::trace;
 use prometheus::IntGauge;
 use std::collections::BTreeSet;
+use std::convert::TryInto;
 use std::time::Duration;
 
 lazy_static::lazy_static! {
     static ref LAST_REFRESH: IntGauge = register_int_gauge!(opts!(
         "zincati_update_agent_last_refresh_timestamp",
         "UTC timestamp of update-agent last refresh tick."
     )).unwrap();
+
+    static ref LOCAL_DEPLOYMENTS_COUNT: IntGauge = register_int_gauge!(
+        "zincati_local_deployments_count",
+        "Total number of local finalized deployments found, excluding the current booted deployment."
```

Comment:
I'm not sure whether there is a better name for those. "Local inactive deployments" maybe?

/cc @jlebon

Reply:
Hmm, do we even need "local"? All deployments are local. :)

Maybe

> Total number of deployments found, excluding any staged and booted one.

?